go.mod

```diff
@@ -1,10 +1,14 @@
 module github.com/openfaas/mqtt-connector
 
-go 1.15
+go 1.18
 
 require (
 	github.com/eclipse/paho.mqtt.golang v1.2.0
 	github.com/openfaas/connector-sdk v0.6.1
 	github.com/openfaas/faas-provider v0.17.1
```

Comment:
fats-provider can be upgraded, as can the connector-sdk. Could you do those too?

Reply:
Every dependency here is now up-to-date.